conduct.md

```diff
@@ -11,7 +11,37 @@ testing fork
 
 * Attendance
 * Deadlines
-* Communication
+
+## Communication
+
+*Team members shall communicate fully, openly, and promptly regarding individual
+comfort level and progress with an assigned task.*
+
+* **Slack shall be the primary method of communication**
+
+* **Acceptable Communication (C):** *Members shall*
+  * Communicate to others when they will be unable to attend any meeting
+  * Communicate about work in progress and personal time constraints.
+  * Share any encountered problem immediately so the team can work toward a solution.
+  * Critique the idea pitched, not the person pitching it.
+
+* *Team Leads Shall*
+  * Give updates on all decision making to keep everyone involved.
+  * Communicate deadlines and expectations to team members.
+  * Meet with team members regularly to assess progress and participation.
+
+* **Unacceptable Communication (U):** *No member shall*
+  * Participate in off task or inappropriate conversations in slack channels.
+  * Criticize other team members for decisions or performance.
+  * Make comments that may be interpreted as abusive or unnecessarily negative.
+  * Make unhelpful and negative comments about members or decisions made.
+
+* *No Team Lead Shall*
+  * Manipulate members to perform their tasks for them
+  * Abuse leadership role to make members complete work unrelated to the class work
+  * Abuse leadership role to berate or act negatively towards team members.
+
+
```

Comment:
Build error:
conduct.md:44: MD012 Multiple consecutive blank lines

Reply:
Just made the change, should build now.